src/components/Header/headerUtils.ts

```diff
@@ -48,3 +49,55 @@ export const parseAlertsData = (data: any): Alert[] => {
 
   return filteredAlerts;
 };
+
+/**
+ * getCookieValue uses the js.cookie package to get the value
+ * of the nyplIdentityPatron cookie (if it exists) and extract
+ * the cookie's access_token.
+ */
+export const getCookieValue = () => {
+  const cookieValue = Cookies.get("nyplIdentityPatron");
+  const accessToken = cookieValue ? JSON.parse(cookieValue).access_token : "";
+
+  return { cookieValue, accessToken };
+};
+
+/**
+ * fetchPatronData uses the patronApiUrl combined with the
+ * access_token from the nyplIdentityPatron cookie to fetch
+ * the patron's information from the server.
+ */
+export const fetchPatronData = (accessToken, cb) => {
+  const patronApiUrl = "https://platform.nypl.org/api/v0.1/auth/patron/tokens/";
+  const fetchErrorMessage =
+    "NYPL Reservoir Header: There was an error fetching NYPL patron data.";
+
+  fetch(`${patronApiUrl}${accessToken}`)
+    .then((response) => {
+      if (response.status >= 200 && response.status < 300) {
+        return response.json();
+      } else {
+        throw new Error(fetchErrorMessage);
+      }
+    })
+    .then(cb)
+    .catch(() => {
+      console.warn(fetchErrorMessage);
+    });
+};
+
+export const extractPatronName = (data) => {
```

Comment:
Add a short comment and give `data` a type, or maybe just leave it as `any` for now.